### ISSUE TITLE
prop-shorthand does not report on _.sortBy

### ISSUE THREAD

**Issue by itamarki** [wix/eslint-plugin-lodash#189]
Hi,

Thank you for this great plugin.

Configuration: canonical.

```
const users = [
    {name: 'Alice', age: 50},
    {name: 'Bob', age: 40}
];`

const sortedUsers = _.sortBy(users, (user) => user.age);
```

Expected:
Warning: "Prefer property shorthand syntax (lodash/prop-shorthand)"
(Because the code can be simplified to `_.sortBy(users, 'age')`)

Actual:
No Warning.

**Comment by ganimomer** [comment]
Hi, and thanks for opening an issue! Fixed in `v4.0.0`.
